src/libs/actions/IOU.ts

```diff
@@ -10905,17 +10908,28 @@ function adjustRemainingSplitShares(transaction: NonNullable<OnyxTypes.Transacti
 /**
  * Put expense on HOLD
  */
-function putOnHold(transactionID: string, comment: string, reportID: string, searchHash?: number) {
+function putOnHold(transactionID: string, comment: string, initialReportID: string | undefined, searchHash?: number) {
```

Comment:
Just a bit more succinct

```suggestion
function putOnHold(transactionID: string, comment: string, initialReportID?: string, searchHash?: number) {
```